keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -205,39 +241,90 @@ internal class ApplicationControllerTests : JUnit5Minutests {
             hasManagedResources: true
             currentEnvironmentConstraints: []
             resources:
-            - id: "ec2:cluster:test:fnord-test"
+            - id: "ec2:cluster:test:fnord-test-west"
               kind: "ec2/cluster@v1"
-              status: "CREATED"
+              status: "HAPPY"
               moniker:
                 app: "fnord"
-                stack: "test"
+                stack: "test-west"
               locations:
                 account: "test"
                 vpc: "vpc0"
                 regions:
                 - name: "us-west-2"
-            - id: "ec2:cluster:test:fnord-staging"
+              artifact:
+                name: "fnord"
+                type: "deb"
+            - id: "ec2:cluster:test:fnord-test-east"
+              kind: "ec2/cluster@v1"
+              status: "HAPPY"
+              moniker:
+                app: "fnord"
+                stack: "test-east"
+              locations:
+                account: "test"
+                vpc: "vpc0"
+                regions:
+                - name: "us-east-1"
+              artifact:
+                name: "fnord"
+                type: "deb"
```

Comment:
Here's where you can see artifact information reflected in the resource summary. This one comes from an image provider that specifies the type and name of the artifact.